### ISSUE TITLE
Dome light rotation + Drivers don't show up

### ISSUE THREAD

**Issue by Anonim001** [bdancer/blender-for-vray#111]
1. Rotation of hemi light doesn't affect sky rotation.
2. Drivers added in node editor for lights, don't show up in Graph Editor.

**Comment by jkntdo** [comment]
You need to connect a Matrix node to the uvm matrix input in order to rotate hdri image in dome light (if that's what you mean)

**Comment by Anonim001** [comment]
The method you described works, but I think that simple rotating the hemi light is quicker and easier than switching from 3D view to node editor for lights and searching a proper node. That's why I tried to add a driver to the matrix node which would be reponsible for changing the value of hdri rotation according to the lamp rotation, Unfortunatelly I can't do that, because the drivers added in light section don't show up in the graph editor.

I've just noticed that drivers added for materials also don't show up in the graph editor.

Cycles:
![cycles](https://cloud.githubusercontent.com/assets/13710958/9477390/5a31a2c8-4b72-11e5-94e2-60f0d376e15c.jpg)

VRay:
![vray](https://cloud.githubusercontent.com/assets/13710958/9477398/60f922fc-4b72-11e5-886a-1b12d127b50d.jpg)
